examples/postgres/advanced-blog-cli/src/main.rs

```diff
@@ -204,7 +206,7 @@ fn convert_auth_error(err: auth::AuthenticationError) -> Box<Error> {
         NoPasswordSet => {
             "No password given. You need to set the BLOG_PASSWORD environment variable.".into()
         }
-        EnvironmentError(e) => e.into(),
+        EnvironmentError(e) => e.compat().into(),
```

Comment:
I may have missed something, but why does this need the failure crate if we're only using it here? Is there a way to do this without the crate?

Just trying to avoid adding another dependency since it doesn't seem to be used in many places.